packages/core/data-transfer/src/strapi/providers/remote-source/index.ts

```diff
@@ -128,13 +140,27 @@ class RemoteStrapiSourceProvider implements ISourceProvider {
 
     // Init the asset map
     const assets: {
-      [filename: string]: IAsset & {
+      // TODO: could we include filename in this for improved logging?
```

Comment:
IAsset does contain the filename, no?

Reply:
Ahhh, maybe it does, I completely missed that it was an IAsset. I might add a log in there in that case